chapters/classes.tex

```diff
@@ -817,9 +817,7 @@ \section{Specialized Classes}\doublelabel{specialized-classes}
 of a record may not have prefixes \lstinline!input!, \lstinline!output!,
 \lstinline!inner!, \lstinline!outer!, \lstinline!stream,!
 or \lstinline!flow!. Enhanced with implicitly available record constructor function,
-see \autoref{record-constructor-functions}. Additionally, record components can be used as
-component references in expressions and in the left hand side of
-assignments, subject to normal type compatibility rules. The components
+see \autoref{record-constructor-functions}. The components
 directly declared in a record may only be of specialized class record
```

Comment:
"record and type" should probably be "record or type".

Reply:
Yes, done!